_codux/boards/clients-section.board.css

```diff
@@ -0,0 +1,85 @@
+.clients {
+    padding: 80px;
+    font-family: Arial, sans-serif;
+
+    row-gap: 60px;
+    display: flex;
+    flex-direction: column;
+    background-color: #ffffff;
+}
+
+.clientsHeader {
+    display: flex;
+
+    @media (width <= 768px) {
+        flex-direction: column-reverse;
+        row-gap: 50px;
+    }
+}
+
+.clientsContentTexts {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.clientsTitle {
+    font-weight: 700;
+    font-size: 36px;
+    line-height: 1.3;
+
+    @media (width <= 768px) {
+        font-size: 22px;
+    }
+}
+
+.clientsDescription {
+    font-size: 16px;
+    line-height: 1.6em;
+    width: 85%;
```

Comment:
this acts as a gap between the description and the title? if yes then just use `gap` on the container.